src/main/java/org/terasology/climateConditions/ClimateConditionsSystem.java

```diff
@@ -15,21 +15,26 @@
  */
 package org.terasology.climateConditions;
 
-import com.google.common.base.Function;
 import com.google.common.collect.Maps;
+import org.terasology.biomesAPI.BiomeRegistry;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.registry.In;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.registry.Share;
+import org.terasology.world.WorldProvider;
+import org.terasology.world.block.BlockManager;
 
 import java.util.Map;
 
 @RegisterSystem(value = RegisterMode.AUTHORITY)
 @Share(value = ClimateConditionsSystem.class)
 public class ClimateConditionsSystem extends BaseComponentSystem {
-    private final float minMultiplier = 0.0005f;
-    private final float maxMultiplier = 0.01f;
+    @In
+    private BlockManager block;
```

Comment:
This naming could probably lead to a fair amount of confusion, and saving a few characters isn't really very important :-) Better it be `blockManager`, same with the `WorldProvider` var below